contrib/kubernetes/static/network-policy.yaml

```diff
@@ -0,0 +1,29 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: sdn
+    namespace: openshift-node
+- apiVersion: authorization.openshift.io/v1
+  kind: ClusterRoleBinding
+  metadata:
+    name: sdn-cluster-reader
+  roleRef:
+    name: cluster-reader
+  subjects:
+  - kind: ServiceAccount
+    name: sdn
+    namespace: openshift-node
+- apiVersion: authorization.openshift.io/v1
```

Comment:
This is in a `kubernetes` directory but using origin authz instead of RBAC?  Unless you have some backwards compatibility use case, please use RBAC.

Reply:
> This is in a kubernetes directory but using origin authz instead of RBAC? Unless you have some backwards compatibility use case, please use RBAC.

What @enj said.  RBAC is source of truth now.

Reply:
This is an example role, not productized yet.

Reply:
> This is an example role, not productized yet.

Yes, but it prevents compatibility with kube without good reason.